### PR TITLE
Add a funding.yml organization-wide

### PR DESCRIPTION
**Pull request by ghost**
Since FUNDING.yml is also a supported [default community health file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file), it should be applied organization-wide instead of repository level, this would also remove the need to have a .github/FUNDING.yml to every repository and in case of adding another supported funding method, it can be added to this file directly instead of updating the files manually

Kinda like #5, this add the [Sponsor this project](https://user-images.githubusercontent.com/95353984/231768590-2e715b5b-1908-458f-934a-098223a0a473.png) area to the side of every repo owned by the organization, before merging this it should be noted that the .github/FUNDING.yml at a repository level will overwrite the organization .github/FUNDING.yml so they should be deleted (for example [apx](https://github.com/Vanilla-OS/apx/blob/main/.github/FUNDING.yml))

### CODE REVIEW

File: FUNDING.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# These are supported funding model platforms
+
+github: vanilla-os
+liberapay: fabricators
```
Comment (kbdharun) [inline FUNDING.yml]:
```suggestion
liberapay: fabricators

```
LGTM (Just this minor suggestion to fix eol), Thanks for your contribution. cc @pietrodicaprio

**Comment by kbdharun** [comment]
The only issue this adds is it will take one extra step to access GitHub sponsors on org level (as most use it over Liberapay in most regions including mine), but this is just a nitpick, displaying both the options is fine as we do in repos.

**Comment by ghost** [comment]
Sure thing for me as github sponsor is also the one that is shown on the website, maybe we can see what @pietrodicaprio thinks about having both options and then decide? Since i just copied what is in the current .github/FUNDING.yml

**Comment by pietrodicaprio** [comment]
The only reason doesn't exist one already at org level is that we tested it in another org without success but the PR is ok for me

**Comment by ghost** [comment]
Gonna try out if it works on a throwaway org tomorrow, marking as draft for now

**Comment by kbdharun** [comment]
> Gonna try out if it works on a throwaway org tomorrow, marking as draft for now

We can't really test it out easily, as the org needs to apply for the GitHub sponsors program too.

Edit. This works and we use the same at tldr-pages Org wide.

**Comment by ghost** [comment]
> We can't really test it out easily, as the org needs to apply for the GitHub sponsors program too.

I probably phrased that badly, i generally meant see if the current funding.yml gets applied organization-wide on another organization and make appropriate tweaks if needed, for github sponsors we can simply remove the `github-sponsors` line temporarely from the file, since we would know it works if the other links work aswell

**Comment by ghost** [comment]
Started experimenting with the funding file in this org: https://github.com/vanillaFundingTest

- The default FUNDING file in the .github seems to be applied correctly
- There are two repos available, one with an overwrite and one that uses the default, the one with the overwrite redirects directly to the github sponsors page, while the other one shows a dialog to choose what method to use.

/cc @kbdharun

**Review by kbdharun — APPROVED** [review]

**Pull request merged**